zapcli/zap_helper.py

```diff
@@ -343,6 +348,170 @@ def html_report(self, file_path):
         report = self.zap.core.htmlreport(apikey=self.api_key)
         self._write_report(report, file_path)
 
+    def generate_config(self, file_path):
```

Comment:
I wonder if it's better to pull most of the config related code for generating and parsing the config file out of this helper, which is more of a wrapper around the API client library. What do you think?